src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -166,6 +166,7 @@ const RenderListFeed = (props) => {
             isSelf={item.anonimity ? false : userId === item?.actor?.id}
             onPressScore={showScoreAlertDialog}
             showScoreButton={showScoreButton}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isShowDM` is being passed to the `RenderListFeed` component, but it's not clear where this prop is coming from. If it's a new prop for the parent component, make sure it's properly defined and passed down. If it's supposed to be derived from some other state or props, ensure that logic is implemented correctly.
```diff
-             isShowDM
+             isShowDM={someConditionOrProp}
```
Please replace `someConditionOrProp` with the actual condition or prop you want to pass as `isShowDM`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->